compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -1033,10 +1033,11 @@ class TreeUnpickler(reader: TastyReader,
          (untpd.Ident(qual.name).withSpan(qual.span), qual.tpe.asInstanceOf[TypeRef])
       }
 
-      def accessibleDenot(pre: Type, name: Name, sig: Signature) = {
-        val d = pre.member(name).atSignature(sig)
+      def accessibleDenot(qualType: Type, name: Name, sig: Signature) = {
+        val pre = ctx.typeAssigner.maybeSkolemizePrefix(qualType, name)
```

Comment:
I'm not super happy with having to recompute whether the prefix needs to be skolemized in the unpickler, but I don't really see a way around it. Maybe asSeenFrom should behave differently when unpickling a tree, but how do you even know if the type on which asSeenFrom is called comes from a tree being unpickled ? And even if we manage to allow unstable prefixes when unpickling, is that really sound when combined with inlining ?